firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -62,34 +67,49 @@ public class ConfigRealtimeHttpClient {
   private final Set<ConfigUpdateListener> listeners;
 
   @GuardedBy("this")
-  private HttpURLConnection httpURLConnection;
+  private Future<?> autoFetchTask;
+
+  @GuardedBy("this")
+  private int RETRIES_REMAINING;
```

Comment:
Uppercase is used only for constant fields.

Reply:
Updated to normal camel case spelling.